packages/analytics/src/analytics.ts

```diff
@@ -0,0 +1,73 @@
+import { encodeDataToQueryString, getRandomId, getUrl } from './utils';
+
+import type { InitOptions, Event, App } from './types';
+
+export class Analytics<T extends Event> {
+    private enabled = false;
+
+    private version: string;
+    private app: App;
+
+    private instanceId?: string;
+    private sessionId?: string;
+    private commitId?: string;
+    private url?: string;
+
+    private callbacks?: InitOptions['callbacks'];
```

Comment:
are these used somewhere? just thinking whether it makes sense to introduce them if not. :thinking:

Reply:
https://github.com/trezor/trezor-suite/blob/9fb769ea65ada9323ef5452458a17643ffbb4cc9/packages/suite/src/actions/suite/analyticsActions.ts#L73

Reply:
You dont need to call more methods on `analytics.disable()` and `analytics.enable()`